bookshop/db/data/sap.capire.bookshop-Contents.csv

```diff
@@ -0,0 +1,64 @@
+ID;parent_ID;name;page;book_ID
```

Comment:
We should not pollute our central getting-started sample, but move that to the fiori / bookstore project.